serve.py

```diff
@@ -7,18 +7,14 @@
 from sanic_cors import CORS
 import requests
 
-GIRDER_URL = os.environ.get("GIRDER_URL", "https://girder.dandiarchive.org").rstrip("/")
+API_URL = os.environ.get("GIRDER_URL", "https://api.dandiarchive.org/api").rstrip("/")
 
-GIRDER_LOCAL_URL = os.environ.get("GIRDER_LOCAL_URL", GIRDER_URL).rstrip("/")
+API_LOCAL_URL = os.environ.get("API_LOCAL_URL", API_URL).rstrip("/")
```

Comment:
I do not see point of having both `API_URL` and `API_LOCAL_URL` -- `API_URL` would pretty much not used.  So why not just to have `API_URL` or did I forget a test scenario where we had to have both for GIRDER?

Reply:
i think this was for CI testing where the local container url could override it. i was just following the same pattern, so if the tests don't need it, we can remove it.